recipes/cfgfile/0.2.8.2.1/conandata.yml

```diff
@@ -1,4 +1,4 @@
 sources:
   "0.2.8.2.1":
     url: "https://github.com/igormironchik/cfgfile/releases/download/0.2.8.2.1/cfgfile-0.2.8.2.1-full.tar.gz"
-    sha256: "a4458949e310c5748114e586fbc7a88df06fb65ecb5baed446d3370a5b67b565"
+    sha256: "6b0fe28323904ceb4567c5f3d450ab6eec6cca5eebad7ad0b77b1967f9e2772d"
```

Comment:
You really should not change releases after publishing, changing the structure might break someone's scripts =)

Reply:
I recreated archive to satisfy your `tools.get()` method. Nothing changed, just removed leading `./` that break your `strip_root=True`.

Reply:
And `cfgfile version 0.2.8.2.1` is a special release, that was created specially for Conan, as PR will be merged no more changes in downloads will be made.